manual/extensions.md

```diff
@@ -34,14 +34,22 @@ See [development](development.md).
 
 * [rubocop-performance](https://github.com/rubocop-hq/rubocop-performance) -
   Performance optimization analysis
-* [rubocop-rails](https://github.com/rubocop-hq/rubocop-rails) - Rails-specific analysis
+* [rubocop-rails](https://github.com/rubocop-hq/rubocop-rails) -
+  Rails-specific analysis
 * [rubocop-rspec](https://github.com/rubocop-hq/rubocop-rspec) -
   RSpec-specific analysis
 * [rubocop-thread_safety](https://github.com/covermymeds/rubocop-thread_safety) -
   Thread-safety analysis
 * [rubocop-require_tools](https://github.com/milch/rubocop-require_tools) -
   Dynamic analysis for missing require statements
-* [rubocop-i18n](https://github.com/puppetlabs/rubocop-i18n) - i18n wrapper function analysis (gettext and rails-i18n)
+* [rubocop-i18n](https://github.com/puppetlabs/rubocop-i18n) -
+  i18n wrapper function analysis (gettext and rails-i18n)
+* [rubocop-sequel](https://github.com/rubocop-hq/rubocop-sequel) -
+  Code style checking for Sequel gem
+* [rubocop-md](https://github.com/rubocop-hq/rubocop-md) -
```

Comment:
rubocop-md is not a custom cop plugin.
So we should add a new section for the plugin, like `Other plugins`, and place the plugin to the section.

Reply:
🤔 Indeed. It adds rubocop functionality to code snippets in md files.

So this would be...? An extension?

Unfortunately we're using the word "extension" to represent all the plugins/add-ons/extensions. The file itself is called `extensions.md`. I think consistent terminology is important. What do you suggest is the best way going forward?

Are you still with `Other plugins`?

I'm just thinking whether we should just keep the plugin like it's now, with the other plugins.

Reply:
Probably other extensions we should have `Additional Cops` and `Other` or something along those lines.

Reply:
Looking at the page something like `### Other Extensions` is what we need. There's also `haml-lint` that's based on RuboCop if I remember correctly.

Reply:
On the other hand - technically those are not plugins/extensions, but related tools. Maybe they don't belong on this page at all. I can't decide on this right now.

Reply:
🤔 I also don't know what would be the best place to put rubocop-md (and haml-lint).

In the meantime, I've updated this PR to only add rubocop-sequel.

I think we can get that one in and maybe rubocop-md after we figure out the best place to put it.

Reply:
Fair enough. Can you remove the unrelated line-wrapping changes that are also part of the commit currently?

Reply:
I added these intentionally, to keep consistency with the rest of the listed extensions. I think we should keep those.

Some of the listed extensions had those lines wrapped, some didn't. Now they all have. Let me know what you think?